READMEs/fixed-rate-exchange-flow.md

```diff
@@ -0,0 +1,162 @@
+<!--
+Copyright 2021 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: Fixed Rate Exchange Flow
+
+It focuses on Alice's experience as a publisher, and Bob's experience as a buyer & consumer.
+
+Here are the steps:
+
+1.  Setup
+2.  Alice creates a data token
+3.  Alice mints & approves data tokens
+4.  Bob buys at fixed rate data tokens
+
+Let's go through each step.
+
+## 1. Setup
+
+### Prerequisites
+
+-   Linux/MacOS
+-   Docker, [allowing non-root users](https://www.thegeekdiary.com/run-docker-as-a-non-root-user/)
+-   Python 3.8.5+
+
+### Run barge services
+
+In a new console:
+
+```console
+#grab repo
+git clone https://github.com/oceanprotocol/barge
+cd barge
+
+#clean up old containers (to be sure)
+docker system prune -a --volumes
+
+#run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
+./start_ocean.sh  --with-provider2
+```
+
+### Run Ocean Market service
+
+In a new console:
+
+```console
+#install
+git clone https://github.com/oceanprotocol/market.git
+cd market
+npm install
+
+#run Ocean Market app
+npm start
+```
+
+Check out the Ocean Market webapp at http://localhost:8000.
+
+Ocean Market is a graphical interface to the backend smart contracts and Ocean services (Aquarius, Provider). The following steps will interface to the backend in a different fashion: using the command-line / console, and won't need Ocean Market. But it's good to understand there are multiple views.
+
+### Install the library
+
+In a new console that we'll call the _work_ console (as we'll use it later):
+
+```console
+#Create your working directory
+mkdir test3
+cd test3
+
+#Initialize virtual environment and activate it.
+python -m venv venv
+source venv/bin/activate
+
+#Install the ocean.py library. Install wheel first to avoid errors.
+pip install wheel
+pip install ocean-lib
+```
+
+### Create config file
+
+In the work console:
+
+```console
+#Create config.ini file and fill it with configuration info
+[eth-network]
```

Comment:
shouldn't there be a more simple config file now? Since we have merged the example_config changes?

Reply:
Integrated Example Config class for a simpler config object.
![Screenshot from 2021-09-01 11-26-54](https://user-images.githubusercontent.com/50501033/131638630-b9a3c853-51b8-40dd-bb71-3c0017621bcc.png)